changelogs/fragments/1954-zos_copy-cr-removal-while-copying-file-to-remote.yml

```
@@ -0,0 +1,3 @@
+minor_changes:
+  - zos_copy - Carriage return will be removed automatically while copying files to remote server.
```

Suggested a description more alike the issue description.

```suggestion
  - zos_script -Support automatic removal of carriage return line breaks [CR, CRLF] when copying local files to USS.
```

---

And I suggest you add another entry in this changelog for bugfixes where you state a bugfix for `zos_copy` that it was only removing the carriage return characters in the first 1024 bytes or first KB.

---

Updated changeling as suggested